packages/ifpack2/src/Ifpack2_BandedContainer_decl.hpp

```diff
@@ -69,8 +66,7 @@ namespace Ifpack2 {
 /// </ol>
 /// BandedContainer stores the diagonal blocks as Banded matrices, and
 /// solves them using either LAPACK (for the four Scalar types that it
```

Comment:
"either" doesn't apply any more to this comment, but we can always fix that later :-)